src/parse.rs

```diff
@@ -61,17 +62,31 @@ pub fn get_file_change_description_from_file_paths(
     }
 }
 
+fn _make_line_number_vec(line: &str) -> Vec<usize> {
+    let mut numbers = Vec::<usize>::new();
+    let pattern = r"[-+]";
+    let re = Regex::new(&pattern).unwrap();
+
+    for s in re.split(line) {
+        let number = s.split(',').nth(0).unwrap().split_whitespace().nth(0);
+        match number {
+            Some(number) => numbers.push(number.parse::<usize>().unwrap()),
+            None => continue,
+        }
+    }
+    return numbers;
+}
+
```

Comment:
Do you think it's possible to do it all with a regular expression, i.e. capture the numbers as well rather than using this function? While we're at it, we could also capture and expose the first set of `@` markers seeing as they will be needed for determining the number of merge parents for #189 cc @rygwdn